bitnami/grafana-operator/values.yaml

```diff
@@ -648,9 +637,6 @@ grafana:
     ## @param grafana.ingress.tls This enables tls support for the ingress resource
     ##
     tls: false
-    ## @param grafana.ingress.tlsSecret The name for the secret to use for the tls termination
-    ##
-    tlsSecret: grafana.local-tls
```

Comment:
Why are we removing this param? It is still used in the `grafana.yaml` template.